tutorials/preprocessing/90_eyetracking_data.py

```diff
@@ -101,123 +108,106 @@
 # coordinates are not reported, and ``"pupil"`` size data are ``0``. We don't want these
-# blink artifacts biasing our analysis, so we have two options: We can either drop the
-# blink periods from our data during epoching, or we can interpolate the missing data
-# during the blink periods. For this tutorial, let's interpolate the missing samples:
+# blink artifacts biasing our analysis, so we have two options: Drop the blink periods
+# from our data during epoching, or interpolate the missing data during the blink
+# periods. For this tutorial, let's interpolate the blink samples:
 
-mne.preprocessing.eyetracking.interpolate_blinks(raw, buffer=0.05)
-# Let's plot our data again to see the result of the interpolation:
-raw.pick(["pupil_right"])  # Let's pick just the pupil channel
-raw.plot(events=events, event_id={"Flash": 3}, event_color="g")
+mne.preprocessing.eyetracking.interpolate_blinks(raw_et, buffer=(0.05, 0.2))
 
 # %%
-# :func:`~mne.preprocessing.eyetracking.interpolate_blinks` performs a simple linear
-# interpolation of the pupil size data during blink periods. the ``buffer`` keyword
-# argument specifies the amount of time (in seconds) before and after the blinks to
-# include in the interpolation. This is helpful because the ``blink`` annotations
-# do not always capture the entire blink in the signal. We specified a value of ``.05``
-# seconds (50 ms), which is slightly more than the default value of ``.025``.
-
-# %%
-# Dealing with high frequency noise
-# ---------------------------------
-# From the plot above, we notice that there is some high frequency noise in the pupil
-# signal. We can remove this noise by low-pass filtering the data:
+# Aligning the eye-tracking data with EEG data
+# --------------------------------------------
+#
+# The eye-tracking data and EEG data were recorded simultaneously, but on different
+# systems, so we'll need to align the data before we can analyze them together. We can
+# do this using the :func:`~mne.preprocessing.realign_raw` function, which will align
+# the data based on the timing of the shared events that are present in both
+# :class:`~mne.io.Raw` objects. We'll use the shared photodiode events we extracted
+# above, but first we need to convert the event onsets from samples to seconds. Once the
+# data have been aligned, we'll add the EEG channels to the eye-tracking raw object.
+
+# Define a few channel groups of interest
+frontal = ["E19", "E11", "E4", "E12", "E5"]
+posterior = ["E61", "E62", "E78", "E67", "E72", "E77"]
+pupil = ["pupil_right"]
+# Convert event onsets from samples to seconds
+et_din_times = et_events[:, 0] / raw_et.info["sfreq"]
+eeg_din_times = eeg_events[:, 0] / raw_eeg.info["sfreq"]
+# Align the data
+mne.preprocessing.realign_raw(
+    raw_et, raw_eeg, et_din_times, eeg_din_times, verbose="error"
+)
+raw_eeg.pick(frontal + posterior)  # so that all channels fit in the plot below
```

Comment:
You can subselect channels in the plot directly instead. Maybe via the order arg?

Reply:
Yeah I looked into this when updating the tutorial - I think the order parameter expects indices and not channel names which I don't find to be very intuitive?

Reply:
Hmm... really `order` should behave like `picks` in principle, but that's a bigger change.

Have you tried `order = pick_channels(raw.ch_names, parietal + occipital, ordered=False)` or so? It will take a list of channel names and give indices back.

Reply:
that does the trick!